### PR TITLE
Fixed issue of peepholeWeight by its pack ordering of 'iof'.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin @fdwr PTAL, thanks.

### CODE REVIEW

**Review by fdwr — APPROVED** [review]
Approved, depending on my comment (whether the spec's example is incomplete, and whether it and the reference should diverge).

File: src/lstm_cell.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export function lstmCell(input, weight, recurrentWeight, hiddenState, cellState,
       add(
           mul(
               cellState,
-              (peepholeWeight ? slice(peepholeWeight, [starts.i], [hiddenSize]) : zero),
+              (peepholeWeight ? slice(peepholeWeight, [0], [hiddenSize]) : zero),
```
[the remark this answers is not in the document]
Comment (BruceDai) [reply src/lstm_cell.js:43]: > It appears `starts.i` is always 0 anyway?

Yes.

> if we're going to use `starts.i` below (e.g. line 47 and line 53), then I'd use `starts.i` here too

Although `starts.i` is equal to **0**, but I think using `starts.i` for `peepholeWeight` would give an illusion that `peepholeWeight` also follows `iofg`/`ifgo` layout, in fact, `peepholeWeight`'s pack ordering is of 'iof'.

Any thoughts? Thanks.

**Review by BruceDai — COMMENTED** [review]
Thanks @fdwr for your reviewing.

File: src/lstm_cell.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (fdwr) [reply src/lstm_cell.js:43]: Okie dokie. For some reason I cannot even resolve my own comments 🙃.

**Review by fdwr — APPROVED** [review]
Still approved.

**Review by huningxin — APPROVED** [review]

**Pull request merged**